geometry_msgs/Pose tcp_location

geometry_msgs/Pose goal_location
